src/material-experimental/mdc-chips/chip-row.ts

```diff
@@ -52,15 +73,45 @@ export class MatChipRow extends MatChip implements AfterContentInit, AfterViewIn
   GridKeyManagerRow<HTMLElement> {
   protected basicChipAttrName = 'mat-basic-chip-row';
 
+  @Input() editable: boolean = false;
+
+  /** Emitted when the chip is edited. */
+  @Output() readonly edited: EventEmitter<MatChipEditedEvent> =
+    new EventEmitter<MatChipEditedEvent>();
+
   /**
    * The focusable wrapper element in the first gridcell, which contains all
    * chip content other than the remove icon.
    */
   @ViewChild('chipContent') chipContent: ElementRef;
 
+  /** The default chip edit input that is used if none is projected into this chip row. */
+  @ViewChild(MatChipEditInput) defaultEditInput?: MatChipEditInput;
+
+  /** The projected chip edit input. */
+  @ContentChild(MatChipEditInput) contentEditInput?: MatChipEditInput;
+
+  /**
+   * Gets the projected chip edit input, or the default input if none is projected in. One of these
+   * two values is guaranteed to be defined.
+   */
+  get editInput(): MatChipEditInput {
```

Comment:
Can this just be a function? We typically use getter/setter for `@Input` but try to avoid it for regular properties since it generates extra code

Reply:
Done. Also made the function private since it's only ever used inside this component.